src/test/run-pass/binding/pat-tuple-7.rs

```diff
@@ -12,6 +12,6 @@
 
 fn main() {
     match 0 {
-        (pat) => assert_eq!(pat, 0)
+        pat => assert_eq!(pat, 0)
```

Comment:
This test may not be serving a purpose anymore with the removal of parentheses. If it should stick around, I can keep or this change or silence the `warning` that now gets generated.